apps/pipeline/upstream/env/aws/secrets-manager.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: aws-secrets
+  namespace: kubeflow
+spec:
+  provider: aws   
+  secretObjects:
+  - secretName: mysql-secret
+    type: Opaque
+    data:
+    - objectName: "user"
+      key: username
+    - objectName: "pass"
+      key: password
+  - secretName: mlpipeline-minio-artifact
+    type: Opaque
+    data:
+    - objectName: "access"
+      key: accesskey
+    - objectName: "secret"
+      key: secretkey
+  parameters:
+    objects: | 
+      - objectName: "rds-secret"
+        objectType: "secretsmanager"
+        jmesPath:
+            - path: "username"
+              objectAlias: "user"
+            - path: "password"
+              objectAlias: "pass"
+      - objectName: "aws-secrets"
```

Comment:
nit: change name to s3 secret

can be done in a separate PR if you want

Reply:
changed to s3-secret